FeatureToggle.Domain/Entity/User Schema/User.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Identity;
+
+namespace FeatureToggle.Domain.Entity.User_Schema
+{
+    public class User : IdentityUser
```

Comment:
What is IdentityUser

Reply:
Its a class of  [Microsoft.AspNetCore.Identity.EntityFrameworkCore].
that already has properties for username, password, mail, etc. 
also has built in password hashing and verifying methods